src/devices/Mcp25xxx/Register/MessageTransmit/TxBxDn.cs

```diff
@@ -60,32 +60,12 @@ public TxBxDn(byte txBufferNumber, int index, byte transmitBufferDataFieldBytes)
         /// </summary>
         /// <param name="address">The address to look up Tx Buffer Number.</param>
         /// <returns>The Tx Buffer Number based on the register address.</returns>
-        public static byte GetTxBufferNumber(Address address)
+        public static byte GetTxBufferNumber(Address address) => address switch
         {
-            switch (address)
-            {
-                case Address.TxB0D0:
-                case Address.TxB0D1:
-                case Address.TxB0D2:
-                case Address.TxB0D3:
-                case Address.TxB0D4:
-                case Address.TxB0D5:
-                case Address.TxB0D6:
-                case Address.TxB0D7:
-                    return 0;
-                case Address.TxB1D0:
-                case Address.TxB1D1:
-                case Address.TxB1D2:
-                case Address.TxB1D3:
-                case Address.TxB1D4:
-                case Address.TxB1D5:
-                case Address.TxB1D6:
-                case Address.TxB1D7:
-                    return 1;
-                default:
-                    throw new ArgumentException($"Invalid address value {address}.", nameof(address));
-            }
-        }
+            Address.TxB0D0 or Address.TxB0D1 or Address.TxB0D2 or Address.TxB0D3 or Address.TxB0D4 or Address.TxB0D5 or Address.TxB0D6 or Address.TxB0D7 => 0,
```

Comment:
this specific case I find old syntax more readable

Reply:
is it possible to specify ranges? (>= && <=)

Reply:
How does this look? https://github.com/dotnet/iot/pull/1303/commits/3516049c0322dec4a5bb9038fecf88cfca842bd9

Reply:
nice :)